### DISCUSSION TITLE
double back to close android app

### DISCUSSION THREAD

**Discussion by brokkolo** [cagnulein/qdomyos-zwift#1384] [Ideas]
Qdomyos-zwift on android closes as soon as I press the back button in the android navigation bar.
When moving between screens it's easy to close the app this way and it will interrupt the recording session (luckily it saves the fit file).
Some apps display a toast message saying something like "press back again to exit the app".
Could this be reasonable?

**Comment by cagnulein** [comment]
yes it makes sense. never happened to me and never had this request in 2 years but it makes sense :)
